serveradmin/serverdb/forms.py

```diff
@@ -1,55 +1,35 @@
 from django import forms
 from django.core.exceptions import ValidationError
-from django.utils.translation import gettext as _
+
+from serveradmin.serverdb.models import ServertypeAttribute
 
 
 class ServertypeAdminForm(forms.ModelForm):
-    def clean(self):
```

Comment:
Is this now safe with the new validators inside the models?

Reply:
Not safe in the models but we just do not allow adding Attributes of type `inet` or `supernet` anymore in the first place therefore this cannot happen.
```
+            self.cleaned_data['attribute'].type in ('inet', 'supernet') and
+            self.instance.servertype.ip_addr_type == 'null'
```